lib/services/platform-environment-requirements.ts

```diff
@@ -136,5 +129,50 @@ export class PlatformEnvironmentRequirements implements IPlatformEnvironmentRequ
 	private fail(message: string): void {
 		this.$errors.fail({ formatStr: message, suppressCommandHelp: true, printOnStdout: true });
 	}
+
+	private getNonInteractiveConsoleMessage(platform: string) {
+		if (this.$nativescriptCloudExtensionService.isInstalled()) {
+			return `${PlatformEnvironmentRequirements.MISSING_LOCAL_SETUP_MESSAGE} To continue, choose one of the following options: ` + EOL
+				+ "Run $ tns setup command to run the setup script to try to automatically configure your environment for local builds." + EOL
+				+ `${this.getCloudBuildsMessage(platform)}` + EOL
+				+ `Verify that your environment is configured according to the system requirements described at ${this.$staticConfig.SYS_REQUIREMENTS_LINK}.`;
+		}
+
+		return `${PlatformEnvironmentRequirements.MISSING_LOCAL_AND_CLOUD_SETUP_MESSAGE}` + EOL
+			+ "Run $ tns setup command to run the setup script to try to automatically configure your environment for local builds." + EOL
+			+ `Run $ tns cloud setup command to install the ${constants.NATIVESCRIPT_CLOUD_EXTENSION_NAME} extension to configure your environment for cloud builds.` + EOL
```

Comment:
this line of the message is the only difference between this message and the one used in the if above. I suggest extrating the cloud message in a const and use it in the message here. This way you'll not duplicate all other messages:
```TypeScript
const cloudBuildMessage = this.$nativescriptCloudExtensionService.isInstalled() ? this.getCloudBuildsMessage(platform) : 
    `Run $ tns cloud setup command to install the ${constants.NATIVESCRIPT_CLOUD_EXTENSION_NAME} extension to configure your environment for cloud builds.`;

```